config/default/manager_pull_policy.yaml

```diff
@@ -8,4 +8,4 @@ spec:
     spec:
       containers:
       - name: manager
-        imagePullPolicy: Always
+        imagePullPolicy: IfNotPresent
```

Comment:
you might have pushed this change by accident

(aside: we really need to find a a way to not modify this file when running e2e...)